FamilySearchIosSampleApp/LoginVC.swift

```diff
@@ -0,0 +1,244 @@
+//
+//  LoginVC.swift
+//  FamilySearchIosSampleApp
+//
+//  Created by Eduardo Flores on 6/3/16.
+//  Copyright © 2016 FamilySearch. All rights reserved.
+//
+
+import UIKit
+
+class LoginVC: UIViewController {
+    
+    @IBOutlet weak var usernameTextFiew: UITextField!
+    @IBOutlet weak var passwordTextField: UITextField!
+    @IBOutlet weak var loginButtonOutlet: UIButton!
+    @IBOutlet weak var activityIndicator: UIActivityIndicatorView!
+    @IBOutlet weak var dataUsageLabel: UILabel!
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        activityIndicator.stopAnimating()
+        
+        usernameTextFiew.placeholder = NSLocalizedString("usernamePlaceholderText", comment: "username, in email form")
+        passwordTextField.placeholder = NSLocalizedString("passwordPlaceholderText", comment: "password")
+        loginButtonOutlet.setTitle(NSLocalizedString("loginText", comment: "text for login button"), forState: UIControlState.Normal)
+        dataUsageLabel.text = NSLocalizedString("loginDataUsage", comment: "description of data usage")
+    }
+
+    override func didReceiveMemoryWarning() {
+        super.didReceiveMemoryWarning()
+        // Dispose of any resources that can be recreated.
+    }
+    
+    @IBAction func loginAction(sender: AnyObject)
+    {
+        activityIndicator.startAnimating()
+        
+        let username = usernameTextFiew.text!
+        let password = passwordTextField.text!
+
+        if ( !(username.isEmpty) && !(password.isEmpty))
+        {
+            // get initial GET call to collections
+            Utilities.getUrlsFromCollections({ (collectionsResponse, error) -> Void in
+                if (error == nil)
+                {
+                    // get the login token
+                    self.getToken(collectionsResponse.tokenUrlString!,
+                        username: username,
+                        password: password,
+                        client_id: AppKeys.API_KEY,
+                        completionToken: {(responseToken, errorToken) -> Void in
+                            if (errorToken != nil)
+                            {
+                                // TODO: handle case when somehow the token is nil
+                            }
+                            else
+                            {
+                                // get user data, with the newly acquired token
+                                self.getCurrentUserData(collectionsResponse.currentUserString!,
```

Comment:
All of these force unwrapped vars is not a good practice we should encourage in our sample app.  We either need to do a `guard` or `if-let` to safely unwrap them or if we are sure that these things will always have values, then let's not make them optional properties of the Links class.  That change would require a new init method on the Links class that would allow you to pass in each of the values when it gets created, but that seems like a cleaner option overall.